packages/browser-destinations/src/destinations/heap/index.ts

```diff
@@ -68,6 +68,13 @@ export const destination: BrowserDestinationDefinition<Settings, HeapApi> = {
         'This is an optional setting used to set the host that loads heap-js. This setting is used when heapJS is self-hosted. In most cased this should be left unset. The hostname should not contain https or app id it will be populated like so: https://${hostname}/js/heap-${appId}.js. For more information visit the heap [docs page](https://developers.heap.io/docs/self-hosting-heapjs).',
       type: 'string',
       required: false
+    },
+    browserArrayLimit: {
+      label: 'Browser Array Limit',
+      description:
+        'This is an optional setting. When set, nested arrays items will be sent in as new Heap events. Defaults to 0.',
```

Comment:
Sorry, I think I had a typo in there
```suggestion
        'This is an optional setting. When set, nested array items will be sent in as new Heap events. Defaults to 0.',
```